server/src/main/java/io/kroki/server/service/Bpmn.java

```diff
@@ -0,0 +1,53 @@
+package io.kroki.server.service;
+
+import io.kroki.server.action.Delegator;
+import io.kroki.server.decode.DiagramSource;
+import io.kroki.server.decode.SourceDecoder;
+import io.kroki.server.error.DecodeException;
+import io.kroki.server.format.FileFormat;
+import io.kroki.server.response.Caching;
+import io.kroki.server.response.DiagramResponse;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.client.WebClient;
+
+import java.util.Collections;
+import java.util.List;
+
+public class Bpmn implements DiagramService {
+
+  private final WebClient client;
+  private final String host;
+  private final int port;
+  private final SourceDecoder sourceDecoder;
+  private final DiagramResponse diagramResponse;
+
+  public Bpmn(Vertx vertx, JsonObject config) {
+    this.client = WebClient.create(vertx);
+    this.sourceDecoder = new SourceDecoder() {
+      @Override
+      public String decode(String encoded) throws DecodeException {
+        return DiagramSource.decode(encoded);
+      }
+    };
+    this.host = config.getString("KROKI_BPMN_HOST", "127.0.0.1");
+    this.port = config.getInteger("KROKI_BPMN_PORT", 8003);
+    this.diagramResponse = new DiagramResponse(new Caching("8.4.8"));
```

Comment:
The version number should be `6.4.2` right?

Reply:
Yes. It is a copy and paste error. I will change it to the correct version.

Reply:
What does version mean in this context? If it is diagramm version then it would be `2.0`.
If it means the version of the underlaying libary it would be `6.4.2`